### DISCUSSION TITLE
Duplicate NAV Frequency settings like COM setting

### DISCUSSION THREAD

**Discussion by NJTravelerDH** [calibx/msfsdeck#201] [Ideas]
Is it possible for you to create a panel to control NAV 1 & NAV 2 frequency settings, just like the one you did for COM frequencies?  I think that almost the same code would work.  Hopefully, that would not be too much work.

**Comment by calibx** [comment]
Hi,
It is planned for next release : https://github.com/calibx/msfsdeck/issues/186

**Comment by calibx** [comment]
2.4.0 is out with the NAV folder !

**Reply by NJTravelerDH** [reply]
Thank you Calibx,
I downloaded 2.4.0 and installed it - AOK.
But I cannot find the NAV folder.  
I have AP, Lights, & ATC, and COm from a previous download, but no NAV folder.  
Do you have a suggestion as to what I need to do

**Reply by calibx** [reply]
Hi,
The NAV folder should be in the "folder" folder on the right part of the UI.
Can you verify the version in the plugin manager then uninstall then reinstall it ?

**Comment by felixfdc** [comment]
THanks a lot for that, @calibx
